lib/ansible/galaxy/dependency_resolution/dataclasses.py

```diff
@@ -39,6 +42,7 @@
 _GALAXY_YAML = b'galaxy.yml'
 _MANIFEST_JSON = b'MANIFEST.json'
 _SOURCE_METADATA_FILE = b'GALAXY.yml'
+_ANSIBLE_PACKAGE_PATH = pathlib.Path(ansible.__file__).parent
```

Comment:
Can we use the relative path to the current module to compute the root? It would let us drop one import:
```suggestion
_ANSIBLE_PACKAGE_PATH = pathlib.Path(__file__).parents[2]
```

Reply:
That would unnecessarily tie the root path calculation to this module, making it less obvious what's happening, and also making it more fragile in the face of refactoring. What benefit is there to dropping the import statement?

Reply:
The module's already long-ago imported, so all we're really incurring here is a sys.modules lookup, and using the root package directly means we can move this module anywhere without breaking the check- hardcoding assumptions about its relationship to the root as you suggest makes the code more brittle for moves.